pkg/dump/meta.go

```diff
@@ -0,0 +1,56 @@
+package dump
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/valyala/fasthttp"
+	"time"
+)
+
+var (
+	version string
+)
+
+const (
+	MetaFilename = "meta.json"
+)
+
+type Meta struct {
+	Version          string `json:"version"`
+	PMMServerVersion string `json:"pmm-server_version"`
+}
+
+func NewMeta(pmmURL string, c *fasthttp.Client) (Meta, error) {
+	statusCode, body, err := c.Post(nil, fmt.Sprintf("%s/v1/Updates/Check", pmmURL), nil)
+	if err != nil {
+		return Meta{}, err
+	}
+	if statusCode != fasthttp.StatusOK {
+		return Meta{}, fmt.Errorf("non-ok status: %d", statusCode)
+	}
+	resp := new(pmmUpdatesResponse)
+	if err := json.Unmarshal(body, resp); err != nil {
+		return Meta{}, fmt.Errorf("failed to unmarshal response: %s", err)
+	}
```

Comment:
move to a separate func - `requestPMMVersion`

this logic must not depend on external things

Reply:
Resolved in 87e1bb75e51f6bdb1da9fe675ca1c023e5356ad0